src/services/BackendService.ts

```diff
@@ -65,7 +65,6 @@ export class BackendService {
             CARTA.EventType.OPEN_FILE,
             CARTA.EventType.OPEN_FILE_ACK,
             CARTA.EventType.RASTER_IMAGE_DATA,
-            CARTA.EventType.REGION_HISTOGRAM_DATA
```

Comment:
sorry this part is deleted due to the log event PR

Reply:
solving merge conflics now